app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/UploadStatus.scala

```diff
@@ -14,6 +14,8 @@ object UploadStatus {
   case object Queued extends UploadStatus("QUEUED")
   case object Processing extends UploadStatus("PROCESSING")
   case object Complete extends UploadStatus("COMPLETE")
+  case object Failed extends UploadStatus("FAILED")
+  case object Aborted extends UploadStatus("ABORTED")
```

Comment:
Going to need to add a migration to update the PostgreSQL enum type this is stored as